### PR TITLE
Update changelog and bump version to v1.6.4

### PR DESCRIPTION
**Pull request by langner**

### CODE REVIEW

**Comment by langner** [comment]
I think Travis is failing for the branch because I rolled back one commit after pushing this PR. Should be good to submit.

File: ANNOUNCE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-On behalf of the cclib development team, we are pleased to announce the release of cclib 1.6.3, which is now available for download from https://cclib.github.io. This is a minor update to version 1.6.1 that includes some new functionality and attributes, as well as bug fixes and small improvements.
+On behalf of the cclib development team, we are pleased to announce the release of cclib 1.6.4, which is now available for download from https://cclib.github.io. This is a minor update to version 1.6.1 that includes some new functionality and attributes, as well as bug fixes and small improvements.
```
Comment (berquist) [inline ANNOUNCE:1]:
```suggestion
On behalf of the cclib development team, we are pleased to announce the release of cclib 1.6.4, which is now available for download from https://cclib.github.io. This is a minor update to version 1.6.3 that includes some new functionality and attributes, as well as bug fixes and small improvements.
```

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,25 @@
+Changes in cclib-1.6.4
+
+Features:
+    * [GSOC2020] New method: bader calculates QTAIM partial charges (Minsik Cho)
+    * [GSOC2020] New methods: bickelhaupt, stockholder, hirshfeld and ddec calculate partial charges (Minsik Cho)
```
Comment (berquist) [inline CHANGELOG:5]:
```suggestion
    * [GSOC2020] New methods: Bader's QTAIM, Bickelhaupt, Stockholder, Hirshfeld, and DDEC6 partial charges (Minsik Cho)
```
and get rid of the first line that only has Bader.

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+    * [GSOC2020] New bridge to Horton (Minsik Cho)
+    * [GSOC2020] Support reading cube files in volume method (Minsik Cho)
+    * New bridge to Atomic Simulation Environment (Felipe S. S. Schneider)
+    * New bridge to PySCF (Amanda Dumi)
+    * New attribute dispersionenergies for molecular dispersion energy corrections (Theo Keane)
```
Comment (berquist) [inline CHANGELOG:10]: This is actually originally from Jonathon for ORCA, and the remaining implementations are from me.

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+    * New attribute vibfconsts for vibrational force constants (Chikashi Shinagawa)
+    * New attribute vibrmasses for vibrational reduced masses (Chikashi Shinagawa)
+    * Support t1_diagnostic in metadata for most parsers (Felipe S. S. Schneider)
```
Comment (berquist) [inline CHANGELOG:13]: This was requested by Felipe but actually implemented by me.

**Review by berquist — CHANGES_REQUESTED** [review]
There's a remaining 1.6.3 in `doc/sphinx/conf.py`.

**Comment by langner** [comment]
> There's a remaining 1.6.3 in `doc/sphinx/conf.py`.

Done.

File: CHANGELOG (hunk 3 above)
Comment (berquist) [reply CHANGELOG]: Ah, it was more that I was expecting no name to be present (following the above convention). Will we have issue filers for major features/bug fixes in the list now? If so, can you add it to https://github.com/cclib/cclib/issues/555?

File: CHANGELOG (hunk 4 above)
Comment (berquist) [reply CHANGELOG:12]: Ok, same response as above.

File: ANNOUNCE [hunk 5]
```diff
@@ -36,12 +36,12 @@ The cclib development team
...
-* New bridge to Psi4 (Felipe S. S. Schneider)
-* New attribute zpve for zero-point vibrational energy correction (kuriba)
-* New attributes for electric transition dipoles of electronic transitions (mwykes)
-* Support ccenergies and mpenergies in ORCA
-* Support grads in MOLCAS (Daniele Padula)
-* Support thermochemistry attributes in GAMESS (Mark Perri)
-* Support Mulliken atomspins in Gaussian (Peter St. John)
-* Support fuzzy matching of attribute in ccget script
-* Many other minor improvements and bug fixes from Felipe S. S. Schneider, Alex Maldonado, James E T Smith, Dustin Wheeler, shijunang, Waylon Peng and others
+* [GSOC2020] New method: bader calculates QTAIM partial charges (Minsik Cho)
```
Comment (berquist) [inline ANNOUNCE:39]: Can you make the same changes here that you made in `CHANGELOG`?

**Review by berquist — CHANGES_REQUESTED** [review]

File: CHANGELOG (hunk 3 above)
Comment (langner) [reply CHANGELOG]: No, you're right... reverted these. Thanks for keeping an eye on these things. Please LMK if I misrepresented something else.

File: CHANGELOG (hunk 4 above)
Comment (langner) [reply CHANGELOG]: Same.

File: ANNOUNCE (hunk 5 above)
Comment (langner) [reply ANNOUNCE:39]: Done, and in the docs changelog.

File: doc/sphinx/changelog.rst [hunk 6]
```diff
@@ -4,6 +4,56 @@
 Changelog
 =========
 
+Changes in cclib-1.6.4
+----------------------
+
+**Features**
+
+* [GSOC2020] New method: bader calculates QTAIM partial charges (Minsik Cho)
```
Comment (berquist) [inline doc/sphinx/changelog.rst:12]: This is a duplicate line with the one below.

File: ANNOUNCE (hunk 5 above)
Comment (berquist) [reply ANNOUNCE:39]: Can you also use
```
[GSOC2020] New methods: Bader's QTAIM, Bickelhaupt, Stockholder, Hirshfeld, and DDEC6 partial charges (Minsik Cho)
```
to replace the top two lines?

**Review by berquist — CHANGES_REQUESTED** [review]

File: ANNOUNCE (hunk 5 above)
Comment (langner) [reply ANNOUNCE]: Done.

File: doc/sphinx/changelog.rst (hunk 6 above)
Comment (langner) [reply doc/sphinx/changelog.rst]: Done.

**Review by berquist — APPROVED** [review]
Haha, sorry about the back and forth. Let's revisit https://github.com/cclib/cclib/issues/618 for 1.6.5.

**Pull request merged**